ISSUE_TEMPLATE.md

```diff
@@ -1,18 +1,20 @@
-For feature requests, please see: http://www.mattermost.org/feature-requests/.
-For troubleshooting questions, please post on the forums: http://forum.mattermost.org/.
+Per Mattermost guidelines, GitHub issues are for bug reports: http://www.mattermost.org/filing-issues/
 
-For issues with the product, please follow the steps at http://www.mattermost.org/filing-issues/ and post here in the following format:
+For troubleshooting see: http://forum.mattermost.org/.
+For feature proposals see: http://www.mattermost.org/feature-requests/
+
+If you've found a bug--obviously unintentional functionality--please report using the following format: 
```

Comment:
I don't like the "obviously" here. It could sound a bit condescending. There's also some behaviour that may not be an obvious bug that's still a bug, like something being misaligned